.github/workflows/release-publish.yml

```
@@ -0,0 +1,54 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/refs/heads/master/src/schemas/json/github-workflow.json
+name: Workflow - Release (publish)
+
+on:
+  release:
+    types: [published]
```

Not sure but I think we can have prereleased event as well
```suggestion
   types: [published, prereleased]
```

---

published include both released and prereleased according to gh docs